engine/unity5/Assets/Scripts/States/MainState.cs

```diff
@@ -162,6 +162,13 @@ public override void Start()
             //If a replay has been selected, load the replay. Otherwise, load the field and robot.
             string selectedReplay = PlayerPrefs.GetString("simSelectedReplay");
 
+            if (PlayerPrefs.GetString("simSelectedRobot", "").Equals(""))
+            {
+                AppModel.ErrorToMenu("ROBOT_SELECT|FIRST");
+                Debug.Log("ASKJHASDFKJHDSAKJHGDSAF");
```

Comment:
Looks good, but what is this debug log for?

Reply:
Honestly can't even remember